src/include/ipc/dai-intel.h

```diff
@@ -83,6 +83,9 @@ struct sof_ipc_dai_ssp_params {
 	uint16_t tdm_per_slot_padding_flag;
 	uint32_t clks_control;
 	uint32_t quirks;
+	uint32_t bclk_delay;	/* guaranteed time (ms) for which BCLK
+				 * will be driven, before sending data
+				 */
```

Comment:
@jajanusz Do we have a way to ensure fields added to end of IPC ABI structs are handled in a reliable way? E.g. when host sends a 4 byte shorter struct, does the FW ensure the "bclk_delay" is initialized to zero?

Reply:
@kv2019i Yes, take a look: https://github.com/thesofproject/sof/blob/master/src/ipc/handler.c#L54

Reply:
@kv2019i That's why you can have `minor` abi changes, when you add something to end and can handle it in a way that value 0 means old-defaults.

Reply:
@jajanusz Ack, ok, excellent! It's very good to have this written down explicitly in the inline docs.